media-libs/smpeg2/smpeg2-2.0.0.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="An MPEG decoding library"
+DESCRIPTION="SMPEG is short for the SDL MPEG library originally developed \
+by Loki Software. SMPEG uses MPEG-1 standard as a video display library, \
+rather than MPEG-2, since only the MPEG-1 standard is unencumbered with \
+software patents in the United States."
+HOMEPAGE="http://www.icculus.org/smpeg"
+COPYRIGHT="1999-2004 Sam Lantinga, Joe Tennies."
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://www.libsdl.org/projects/smpeg/release/smpeg2-$portVersion.tar.gz"
+CHECKSUM_SHA256="979a65b211744a44fa641a9b6e4d64e64a12ff703ae776bafe3c4c4cd85494b3"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	smpeg2$secondaryArchSuffix = 2.0.0 compat >= 2
+	cmd:smpeg2 = 2.0.0 compat >= 2
+	cmd:plaympeg$secondaryArchSuffix = 2.0.0 compat >= 2
+	lib:libsmpeg2$secondaryArchSuffix = 2.0_0.0.0 compat >= 2.0_0
```

Comment:
The created lib is called "libsmpeg2-2.0.so.0.0.0", should I alter this as?:
lib:libsmpeg2$secondaryArchSuffix = 0.0.0 compat >= 0
PS, in the sdl libs I see more "_" in the recipes?